cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceProgramRunner.java

```diff
@@ -178,4 +181,22 @@ public ProgramController run(Program program, ProgramOptions options) {
     }
     return controller;
   }
+
+  @Nullable
```

Comment:
Ah looks like it could be use by IntelliJ Idea to check if method could return null.